CHANGES.rst

```diff
@@ -5,6 +5,38 @@ Release Notes
 This is the list of changes to scikit-build between each release. For full
 details, see the commit logs at http://github.com/scikit-build/scikit-build
 
+
+Scikit-build 0.16.0
+===================
+
+This release adds support for Python 3.11 and removes support for Python 2.7
+and 3.5 (:pr:`688`). Testing and static checking improved, including being
+fully statically typed internally (though setuptools is not fully typed, so
+it's of limited use).
+
+All deprecated setuptools/distutils features are also deprecated in
+scikit-build, like the ``test`` command, ``easy_install``, etc. Editable mode
+is still unsupported.  Python 3.6 support is deprecated. Older versions of
```

Comment:
Wondering if explicitly adding `Known issues` section would help

Reply:
Not a bad idea; I added a section in the readme.